cmd/utils/executor/executor.go

```diff
@@ -75,6 +76,18 @@ func (e *Executor) Execute(cmdInfo model.DeployCommand, env []string) error {
 	return err
 }
 
+// getCMD returns the command to be executed based on the OS
+// We needed to diferentiate between windows and other OS because we were having problems
+// with using bash on windows: see:
+// - https://github.com/okteto/okteto/issues/2820
+// - https://github.com/okteto/okteto/issues/2592
+func getCMD(command string) *exec.Cmd {
```

Comment:
`getCMDByOS` I think it is more specific. WDYT?

Reply:
outdated comment